rolling/distribution.yaml

```diff
@@ -3208,6 +3208,12 @@ repositories:
       url: https://github.com/adityapande-1995/linux_isolate_process.git
       version: main
     status: maintained
+  load_balancing_service:
```

Comment:
just a checking. currently this package can support only rolling? @Barry-Xu-2018

Reply:
Yes. This package depends on `class GenericService`  which is only implemented in rolling.